source/iopipe/json/parser.d

```diff
@@ -1957,9 +1957,23 @@ struct JSONTokenizer(Chain, ParseConfig cfg)
      */
     JSONToken skipItem()
     {
-        size_t depth = 0;
-        // parse until we see the stack length get less than our current depth,
-        // until we see a comma, error, or end of stream.
+        auto token = leaveNestingLevel(0);
+        if(token == JSONToken.Error)
+            return token;
+        return peekSignificant();
+    }
+    
+
+    /** Seek until leaving the current object or array. 
+     * When next item will be ObjectStart or ArrayStart, will behave similar to JSONItem.skipItem(), but without consuming the ArrayEnd/ObjectEnd
```

Comment:
This description seems wrong and/or confusing.

If I follow the implementation, the function will leave `depth` levels of aggregates, returning the final token it consumed.

I don't think there's any reason to define this in terms or contrast with `skipItem`.